docs/spec/governance/state.md

```diff
@@ -77,37 +87,34 @@ type Proposal struct {
   TotalDeposit          sdk.Coins           //  Current deposit on this proposal. Initial value is set at InitialDeposit
   Deposits              []Deposit           //  List of deposits on the proposal
   SubmitBlock           int64               //  Height of the block where TxGovSubmitProposal was included
-  Submitter             crypto.Address      //  Address of the submitter
+  Submitter             sdk.Address      //  Address of the submitter
   
   VotingStartBlock      int64               //  Height of the block where MinDeposit was reached. -1 if MinDeposit is not reached
-  InitTotalVotingPower  int64               //  Total voting power when proposal enters voting period (default 0)
   InitProcedure         Procedure           //  Active Procedure when proposal enters voting period
+  CurrentStatus         ProposalStatus      //  Current status of the proposal
 
-  Votes                 Votes               //  Total votes for each option
+  YesVotes              sdk.Rat
+  NoVotes               sdk.Rat
+  NoWithVetoVotes       sdk.Rat
+  AbstainVotes          sdk.Rat
 }
 ```
 
-We also introduce a type `ValidatorGovInfo`
+We also mention a method to update the tally for a given proposal:
 
 ```go
-type ValidatorGovInfo struct {
-  InitVotingPower     int64   //  Voting power of validator when proposal enters voting period
-  Minus               int64   //  Minus of validator, used to compute validator's voting power
-}
+  func (proposal Proposal) updateTally(vote byte, amount sdk.Rat)
 ```
 
 ### Stores
 
-*Stores are KVStores in the multistore. The key to find the store is the first parameter in the list*
+*Stores are KVStores in the multistore. The key to find the store is the first parameter in the list*`
+
+We will use one KVStore `Governance` to store two mappings:
 
+* A mapping from `proposalID` to `Proposal`
```

Comment:
Can we make this "from `proposalID:proposal`" so it makes getting the subspace iterator for votes easier?

Reply:
ok